Makefile

```diff
@@ -1,4 +1,5 @@
 
+BROWSERIFY := node_modules/.bin/browserify
```

Comment:
@boneskull why? To me this still seems more straightforward than most alternatives (which admittedly says a lot about the alternatives as well..)